README.md

```diff
@@ -93,16 +93,28 @@ Parameters are :
      - it's used to retrieve informations from selectors
    - **prefix** (optional) is added to actions to avoid some collisions when there are two reducers with the same name in two distincts paths
    - **type** (optional) can be `keyValue` or `simpleObject` (default is `keyValue`)
-   - **defaultData** (optional), for `simpleObject` only, set the default data value, used by `reset` action and for initialisation (default is an empty object `{}`)
+   - **defaultData** (optional), set the default data value, used by `reset` action and for initialisation (default is an empty object `{}` for `simpleObject` and default hashmap model for `keyValue` - see [keyValue types section](./TYPES.md#state))
 
 Example:
  - this reducer will use `id` as key field
  - it's combined into `state.api.raw`
- - its name is `todos`
+ - it's name is `todos`
+ - have default data
 ```es6
 import factory from 'k-redux-factory'
 
-export default factory({ key: 'id', path: 'api.raw', name: 'todos' })
+const defaultData = [
```

Comment:
I would remove this from this section since this is a teaser of the library.
With this code (defaultData) it seems to be a bloated API (too much line of codes)